### PR TITLE
spell change

### PR DESCRIPTION
**Pull request by sroycho4**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 > 🚀 We are granting pilot access to **Ivy\'s Tracer and Transpiler**
 > to some users, [join the waitlist](https://console.unify.ai/) if you
-> want to test them out!
+> want to test them out follow the steps given below!
```
Comment (KareemMAX) [inline README.md:3]: The steps are not described "below", the primary call of action is for the user to visit the console

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -9,9 +9,9 @@ Please, follow the next process when you work on your subtask:
 1. **Choosing a Task:**
 
    - Choose a task to work on which:
-     - is not marked as completed with a tick.
-     - does not have an issue created.
-     - is not mentioned in the comments.
+     - Is not marked as completed with a tick.
+     - Does not have an issue created.
+     - Is not mentioned in the comments.
```
Comment (KareemMAX) [inline CONTRIBUTING.md:14]: I'm not an English expert, but I don't think capitalizing the words is right

File: docs/overview/contributing/building_the_docs.rst [hunk 3]
```diff
@@ -107,3 +107,4 @@ Then, run the following command to build the docs:
 The script will install the required dependencies for `sphinx <https://www.sphinx-doc.org>`_
 which is used to build the docs, as well as dependencies required by Ivy. Then it will
 build the docs for Ivy and store it in ``docs/build``.
+Hope this will help!
```
Comment (KareemMAX) [inline docs/overview/contributing/building_the_docs.rst:110]: I don't think this is necessary

**Review by KareemMAX — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (sroycho4) [reply CONTRIBUTING.md:14]: @KareemMAX , Its a basic rule, First letter of any sentence is always capital

**Pull request closed without merging**